src/teb_local_planner_ros.cpp

```diff
@@ -903,6 +905,15 @@ void TebLocalPlannerROS::saturateVelocity(double& vx, double& vy, double& omega,
     vy *= ratio_y;
     omega *= ratio_omega;
   }
+
+  max_vel_linear = std::max(std::max(max_vel_x, max_vel_y), max_vel_linear);
+  double vel_linear = std::sqrt(vx*vx + vy*vy);
+  if (vel_linear > std::abs(max_vel_linear))
```

Comment:
```suggestion
  if (vel_linear > max_vel_linear)
```
F: no need for `abs` since this willl always be >= 0